### ISSUE TITLE
wishbone_gui.py in Ubuntu

### ISSUE THREAD

**Issue by peterch405** [ManuSetty/wishbone#18]
I can't seem to get this working. The notebook examples work.

```
wishbone_gui.py
Traceback (most recent call last):
  File "/usr/local/bin/wishbone_gui.py", line 5, in <module>
    from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2TkAgg
  File "/usr/local/lib/python3.4/dist-packages/matplotlib/backends/backend_tkagg.py", line 13, in <module>
    import matplotlib.backends.tkagg as tkagg
  File "/usr/local/lib/python3.4/dist-packages/matplotlib/backends/tkagg.py", line 9, in <module>
    from matplotlib.backends import _tkagg
ImportError: cannot import name '_tkagg'
```

`python -c 'import matplotlib as mpl; print(mpl.get_backend())'`

Produces:
`TkAgg`

So I think I have the correct backend...

**Comment by pkathail** [comment]
Try installing tkinter, then uninstalling and reinstalling matplotlib with the following commands to make sure the tkinter files link properly. Let me know if you still run into problems!

`sudo apt-get install tk`
`sudo apt-get install tk-dev`
`pip3 uninstall -y matplotlib`
`pip3 --no-cache-dir install -U matplotlib`

**Comment by peterch405** [comment]
I already had tk and tk-dev installed. Ran all the commands just in case. Still getting the error.

**Comment by pkathail** [comment]
What versions of Ubuntu, matplotlib, and python are you using?

**Comment by peterch405** [comment]
```
lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.4 LTS
Release:	14.04
Codename:	trusty
```

```
python --version
Python 3.4.3
```

```
>>> import matplotlib
>>> matplotlib.__version__
'1.5.1'
```

**Comment by pkathail** [comment]
I tried recreating the error with the versions you specified, but was unable to. The suggestion I posted above is what worked for me when I got the same error, but with different Ubuntu/python versions (Ubuntu 16.04 & Python 3.5).

There are a few other methods for how to solve it here - http://stackoverflow.com/questions/32188180/from-matplotlib-backends-import-tkagg-importerror-cannot-import-name-tkagg

You might also want to try uninstalling and reinstalling wishbone to see if that helps.
